### ISSUE TITLE
[Feature Request] Option to disable notifications

### ISSUE THREAD

**Issue by chuckharmston** [getmatterapp/obsidian-matter#33]
They can be pretty noisy, especially on mobile, and some of them are usually not especially useful (e.g. "Syncing with Matter"). Perhaps a setting with three options:

- All notifications
- Only errors
- No notifications

Thanks for your consideration!

**Comment by MDsza** [comment]
Yes, please add the option to disable the quite noisy notifications on every start of the app on iOS!

Thanks in advance.

**Comment by hunterclarke** [comment]
Released in 1.1.4 thanks to @ryangomba
